tjpcov/covariance_cluster_counts.py

```diff
@@ -0,0 +1,117 @@
+from .covariance_clusters import CovarianceClusters
+import numpy as np
+import pyccl as ccl
+from scipy.integrate import romb
+
+
+class ClusterCounts(CovarianceClusters):
+    """Implementation of cluster covariance that calculates the autocorrelation
+    of cluster counts (NxN)
+    """
+
+    cov_type = "fourier"
+    _reshape_order = "F"
+    _tracer_types = ("cluster", "cluster")
+
+    def __init__(self, config):
+        """Concrete implementation of covariance of cluster counts,
+        specifically the number count auto-correlation.
+
+        Args:
+            config: The configuration file path used to calculate the
```

Comment:
Check the `rtd` branch. To avoid having to change it later, put this here
```
        Args:
            config (dict or str): If dict, it returns the configuration
                dictionary directly. If string, it asumes a YAML file and
                parses it.
```

Reply:
Got it - I actually opted to not include types in my documentation but I can include them.

Reply:
I was following what they did for CCL.